src/js/base/logic.js

```diff
@@ -268,3 +268,269 @@ ripe.Ripe.prototype.localeModel = function(
         fallback: fallback
     });
 };
+
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype._toLocale = function(
+    value,
+    brand,
+    model,
+    owner,
+    {
+        locale = null,
+        defaultValue = null,
+        prefix = "builds",
+        fallback = false,
+        compatibility = true,
+        hack = false
+    } = {}
+) {
+    const values = Array.isArray(value) ? value : [value];
+    const result = this.localeModel(value, owner, {
+        brand: brand,
+        model: model,
+        locale: locale,
+        defaultValue: defaultValue,
+        prefix: prefix,
+        fallback: fallback,
+        compatibility: compatibility,
+        hack: hack
+    });
+
+    // if the localization was successful returns its result
+    if (result !== defaultValue) return result;
+
+    // if the localization was not successful but a default
+    // was defined, then returns it
+    if (defaultValue !== null) return result;
+
+    // otherwise run the localization process again using
+    // a fallback locale as the base for localization
+    const localeFallback = owner.getLocaleFallback();
+    if (localeFallback !== locale) {
+        return this._toLocale(values, brand, model, {
+            locale: localeFallback,
+            defaultValue: defaultValue,
+            prefix: prefix,
+            fallback: fallback,
+            compatibility: compatibility,
+            hack: hack
+        });
+    }
+
+    return values[0];
+};
+
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype._localeModel = function(

```

Comment:
Delete. This already exists

Reply:
As the above comment, it's not completly the same, as it uses _toLocale and that:
https://github.com/ripe-tech/ripe-sdk/pull/252#discussion_r580112728